canonical/canonical.com
# Include departments with no vacancies

## Done

- Fixed missing departments
- Fixed 404ing career-explorer
- Removed links to working here pages

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: https://canonical-com-779.demos.haus/careers
- See that Administration and Legal show up in overview and take you to the correct dept page
- View the site locally in your web browser at: https://canonical-com-779.demos.haus/careers/career-explorer
- See that the page renders successfully 

## Issue / Card

Fixes # https://warthogs.atlassian.net/browse/WD-1734
